builtin/hash-object.c

```diff
@@ -100,7 +97,8 @@ int cmd_hash_object(int argc, const char **argv, const char *prefix)
 			HASH_WRITE_OBJECT),
 		OPT_COUNTUP( 0 , "stdin", &hashstdin, N_("read the object from stdin")),
 		OPT_BOOL( 0 , "stdin-paths", &stdin_paths, N_("read file names from stdin")),
-		OPT_BOOL( 0 , "no-filters", &no_filters, N_("store file as is without filters")),
+		OPT_BIT(0 , "no-filters", &flags, N_("store file as is without filters"),
```

Comment:
Style is:
```suggestion
		OPT_BIT( 0 , "no-filters", &flags, N_("store file as is without filters"),
```
to pad where the `'`s go for short options, at least in this file.

Reply:
Grepping `buildins/*.c` shows that `(0` is used at least 689 times, while `( 0` at most 23.

The coding guidelines don’t mention any rule mandating this space, but say: *Use whitespace around operators and keywords, but not inside parentheses and not around functions.*

Reply:
However, I forgot to delete the other space.